workers/ohsome_quality_analyst/oqt.py

```diff
@@ -32,41 +37,53 @@
 from ohsome_quality_analyst.utils.helper import loads_geojson, name_to_class
 
 
-async def create_indicator_as_geojson(
-    parameters: Union[IndicatorBpolys, IndicatorDatabase],
-    force: bool = False,
+@singledispatch
+async def create_indicator_as_geojson(parameters):
+    raise NotImplementedError(
+        "Cannot create Indicator as GeoJSON for parameters of type: "
+        + str(type(parameters))
+    )
+
+
+@create_indicator_as_geojson.register(IndicatorBpolys)
+@create_indicator_as_geojson.register(IndicatorData)
+async def _create_indicator_as_geojson(  # noqa
+    parameters: Union[IndicatorBpolys, IndicatorData],
     size_restriction: bool = False,
+    **_kargs,
 ) -> Union[Feature, FeatureCollection]:
     """Create an indicator or multiple indicators as GeoJSON object.
 
     Returns:
         Depending on the input a single indicator as GeoJSON Feature will be returned
         or multiple indicators as GeoJSON FeatureCollection will be returned.
     """
-    if isinstance(parameters, IndicatorBpolys):
-        features = []
-        for i, feature in enumerate(loads_geojson(parameters.bpolys)):
-            if "id" in feature.keys():
-                id_ = str(feature["id"])
-            else:
-                id_ = str(i)
-            logging.info("Input feature identifier: " + id_)
-            if size_restriction:
-                await check_area_size(feature.geometry)
-            indicator = await create_indicator(
-                parameters.copy(update={"bpolys": feature}),
-                force,
-            )
-            features.append(indicator.as_feature(flatten=True))
-        if len(features) == 1:
-            return features[0]
+    features = []
+    for i, feature in enumerate(loads_geojson(parameters.bpolys)):
+        if "id" in feature.keys():
+            id_ = str(feature["id"])
         else:
-            return FeatureCollection(features=features)
-    elif isinstance(parameters, IndicatorDatabase):
-        indicator = await create_indicator(parameters, force)
-        return indicator.as_feature(flatten=True)
+            id_ = str(i)
+        logging.info("Input feature identifier: " + id_)
+        if size_restriction:
+            await check_area_size(feature.geometry)
+        indicator = await create_indicator(parameters.copy(update={"bpolys": feature}))
+        features.append(indicator.as_feature(flatten=True))
+    if len(features) == 1:
+        return features[0]
     else:
-        raise ValueError("Unexpected parameters: " + str(parameters))
+        return FeatureCollection(features=features)
+
+
+@create_indicator_as_geojson.register
+async def _create_indicator_as_geojson(  # noqa
```

Comment:
see above

Reply:
There is. Added exact flake8 warning code.